### ISSUE TITLE
VerticalTabs: <li> contains <div> which violates accessibility requirement of semantic HTML

### ISSUE THREAD

**Issue by mdamian322** [pluralsight/design-system#1963]
### Desired behavior
divs should not be inside list items

### Today's behavior
`div>` is inside of `<li>` element [here](https://github.com/pluralsight/design-system/blob/b5abe62c164f47cde093b6fce1381af2cb47e21e/packages/verticaltabs/src/react/item.tsx#L147)
![Screen Shot 2021-07-28 at 2 38 29 PM](https://user-images.githubusercontent.com/13877138/129630913-1a58907b-c0f8-4912-a5da-7f230a400016.png)

### The Value Add
Can make the semantic HTML better for accessibility
See this [article](https://app.pluralsight.com/guides/create-ordered-lists-with-html5-screen-readers ) under "Accessible Lists in React" section

**Comment by mdamian322** [comment]
Let me know when this is fixed so we can then update the package in our repo. Thanks!

**Comment by EdwardIrby** [comment]
@mdamian322 This is not invalid. LI's can contain flow content (div) according to the spec especially in HTML5 which is supported in all modern browsers. We recently dropped IE11 support. 

We won't fix this

Please see
- https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element
- https://stackoverflow.com/a/18269134
- https://stackoverflow.com/questions/1915077/is-div-inside-list-allowed
- https://stackoverflow.com/questions/6449772/can-i-use-a-div-inside-a-list-item
